app/grandchallenge/challenges/emails.py

```diff
@@ -1,6 +1,8 @@
+from django.contrib.auth.models import Group
 from django.contrib.sites.models import Site
-from django.core.mail import mail_managers
+from django.core.mail import mail_managers, send_mail
 
+from config import settings
```

Comment:
Do not import settings from config, use `from django.conf import settings`. Maybe we should add a pre-commit check for this?